exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -136,13 +136,13 @@ func (e *exporter) serializeMetrics(md pdata.Metrics) []string {
 				metricLines, err := serialization.SerializeMetric(e.settings.Logger, e.cfg.Prefix, metric, e.defaultDimensions, e.staticDimensions, e.prevPts)
 
 				if err != nil {
-					e.settings.Logger.Sugar().Errorf("failed to serialize %s %s: %s", metric.DataType().String(), metric.Name(), err.Error())
+					e.settings.Logger.Sugar().Errorf("failed to serialize %s %s", metric.DataType().String(), metric.Name(), zap.Error(err))
```

Comment:
@MovieStoreGuy can I still use the structured logging `zap.Error` with `Sugar().Errorf`?

Reply:
Nevermind